js/about/preferences.js

```diff
@@ -365,11 +365,12 @@ class GeneralTab extends ImmutableComponent {
 
 class SearchSelectEntry extends ImmutableComponent {
   shouldComponentUpdate (nextProps, nextState) {
-    return this.props.settings.get(settings.DEFAULT_SEARCH_ENGINE) !== nextProps.settings.get(settings.DEFAULT_SEARCH_ENGINE)
+    return getSetting(settings.DEFAULT_SEARCH_ENGINE, this.props.settings) !==
+      getSetting(settings.DEFAULT_SEARCH_ENGINE, nextProps.settings)
```

Comment:
I commented on the commit on your branch by mistake instead of here:
https://github.com/darkdh/browser-laptop/commit/f82a848a08780ff0b2f4ded95b2000514f937323